app/services/doc_auth/lexis_nexis/request.rb

```diff
@@ -123,9 +125,16 @@ def settings
         }
       end
 
-      # AM: Need to account for the uuid-prefix when folding in the lexisnexis gem.
       def uuid
-        SecureRandom.uuid
+        return SecureRandom.uuid unless applicant
```

Comment:
where does `applicant` come from here?

Reply:
it doesn't - I missed it in making the changes from `applicant` to `user_uuid`/`uuid_prefix` ;-) That's what I get making changes in between meetings...